core/src/bundle/usBundleEvent.cpp

```diff
@@ -65,7 +65,7 @@ bool BundleEvent::IsNull() const
   return !d;
 }
 
-BundleEvent::BundleEvent(Type type, Bundle* bundle)
+BundleEvent::BundleEvent(Type type, std::shared_ptr<Bundle> bundle)
```

Comment:
why is this not const reference ? All its doing is passing the param onto the constructor of BundleEventData which should make a copy of the shared_ptr if it wants shared ownership.